### PR TITLE
Create the home page (Html file)

### LINKED ISSUE #4: Home Page
- ## Main purpose:
The home page which giueds you to the rest of the project.

- ## Main tasks:

-  **Create HTML file with the following section which contains :**
          1. Image
          2. paragraph.
          3. Button.
     
 -   **Create CSS file with the following:**
         1. Styling the image.
         2. Styling the paragraph.
         3. Styling the button.
 

- ## UI design:

### PR DESCRIPTION
**Pull request by Maramfathi99**
Relates #4
This page contains the HTML structure.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Booklopedia</title>
+    <link rel="stylesheet" href="style.css" />
```
Comment (sara219) [inline index.html]: your file name is `home.css` not `style.css`

```suggestion
    <link rel="stylesheet" href="home.css" />
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=ABeeZee&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <section class="main-page">
+      <img src="Component.png" />
```
Comment (sara219) [inline index.html]: the path for the image is not right.
the component image is inside the images folder, not in the root as the HTML file.

```suggestion
      <img src="./image/Component.png" alt="" />
```

Screenshot from 2021-04-13 22-34-30

File: home.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height:667 ;
+}
+img {
+  width: 375px;
+  height: 233px;
```
Comment (sara219) [inline home.css]: the picture has both height and width set, which distorts it a little. I think you should select only the width

Screenshot from 2021-04-13 22-55-42

File: home.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
+*{
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+    background: #191927;
```
Comment (sara219) [inline home.css:5]: you can clearly see the color difference between the background of the image and the other color

Screenshot from 2021-04-13 23-00-42

use `#211c2c`
```suggestion
    background: #211c2c;
```

**Review by sara219 — CHANGES_REQUESTED** [review]

**Review by sara219 — APPROVED** [review]
💯

**Review by Raghad-Mezied — APPROVED** [review]

**Pull request merged**